modules/Campaigns/ProcessBouncedEmails.php

```diff
@@ -101,20 +107,58 @@ function createBouncedCampaignLogEntry($row, $email, $email_description)
 }
 
 /**
- * Given an email address, mark it as invalid.
+ * Given the email description returns if the email should be marked invalid.
  *
  * @param $email_address
+ * @return bool
  */
-function markEmailAddressInvalid($email_address)
```

Comment:
I understand if you change the function name it makes more logical and readable the code itself but this change could brakes the backward compatbility to any customization which already exists and possible usage of current exists functions.

Reply:
ok, I've left the old function as is an made the new one call the old one.